html/about-page.html

```diff
@@ -0,0 +1,177 @@
+             exploring the GitHub tools, techniques, and best practices for securing your software development lifecycle.</p>
+            </div>
+        <p class="contact"> Please contact us on email if you have any query </p>
+        <a href="#" class="email"> abdulahadb802@gmail.com <img src="../images/arrow-30-16.png" alt="arrow "></a>
+            </div>
+        </section>
+        <section class="container-fluid logo-history">
+            <div class="main container">
+                <h3 class="About-title">Creative GitHub Universe logo</h3>
+                <span class="line"></span>
+                <p class="discuss"> Github logo is well know by every coder and is created by CEO of Github</p>
+                <div class="logo1">
+                  <a href=""><img src="../images/github-9-128.png" alt="github pic">Universe 2022</a>
+              </div>
+                </div>
+        </section>
+        <section class="container-fluid">
+          <div class="main container">
+              <h3 class="About-title">See Past GitHub Universe Summit</h3>
+              <span class="line"></span>
+              <p class="discuss"> Github logo is well know by every coder and is created by CEO of Github</p>
+              <div class="history">
+              <div class="poti">
+                <div class="card">
+                  <div class="card-body">
+                      <img class="" src="../images/speaker1.jpg" alt="aaron francis speaker image">
+                    <div class="poti2">
+                    <h5 class="year">2013</h5>
+                    <h6 href="" class="disc">GitHub Universe in 2013</h6>
+                  </div>
+                </div>
+              </div>
+              </div>
+              <div class="poti">
+                <div class="card">
+                  <div class="card-body">
+                      <img class="" src="../images/speaker1.jpg" alt="aaron francis speaker image">
+                    <div class="poti2">
+                    <h5 class="year">2013</h5>
+                    <h6 href="" class="disc">GitHub Universe in 2013</h6>
+                  </div>
+                </div>
+              </div>
+              </div>
+              </div>
+      </section>  
+      <section class="container-fluid pg3 ">
+        <div class="main">
+        <h3 class="main-title1">Partner</h3>
+        
+        <span class="line"></span>
+        
+        <div class="container Program-content1">
+            <div>    
+            <ul>
+                    <li>
+                        <img src="../images/js-128.png" alt=" ">
```

Comment:
- [ ] Kindly, don't forget to add the `alt` property for all of your images 🎃, on this way you are writing code with the best practices. 💯